app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.party_player">
+    package="com.example.partyPlayer">
```

Comment:
Changed the package names by removing underscores to comply with Kotlin naming conventions stated here: https://kotlinlang.org/docs/reference/coding-conventions.html